src/nomad_simulations/properties/energies.py

```diff
@@ -77,3 +78,502 @@ def __init__(
+    """
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+
+class DihedralEnergy(ClassicalEnergy):
+    """
+    Physical property section describing contributions to the potential energy from dihedral interactions of a (sub)system.
+    """
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+
+class ImproperDihedralEnergy(ClassicalEnergy):
+    """
+    Physical property section describing contributions to the potential energy from improper dihedral interactions of a (sub)system.
+    """
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+
+class ExternalEnergy(ClassicalEnergy):
+    """
+    Physical property section describing contributions to the potential energy from external interactions of a (sub)system.
+    """
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+######################################
+# List of quantum energy contributions
+######################################
+
+class ElectronicEnergy(QuantumEnergy):
+    """
+    Physical property section describing the electronic energy of a (sub)system.
+    """
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+#! allowed_contributions = ['PotKin'] Not sure how to deal with sub-contributions... - I lean towards keeping a flat list but I am not sure of all the usages
+
+
+class ElectronicKineticEnergy(QuantumEnergy):
+    """
+    Physical property section describing the electronic kinetic energy of a (sub)system.
+    """
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+
+class XCEnergy(QuantumEnergy):
+    """
+    Physical property section describing the exchange-correlation (XC) energy of a (sub)system,
+    calculated using the functional stored in XC_functional.
+    """
+    # ! Someone check this description!
+    # ? Do we really want to specify the method here? This can't be user-defined?
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+
+class XCPotentialEnergy(QuantumEnergy):
+    """
+    Physical property section describing the potential energy contribution to the exchange-correlation (XC) energy,
+    i.e., the integral of the first order derivative of the functional
+        stored in XC_functional (integral of v_xc*electron_density), i.e., the component
+        of XC that is in the sum of the eigenvalues. Value associated with the
+        configuration, should be the most converged value.          |
+    """
+    # ! Someone check this description!
+    # ? Do we really want to specify the method here? This can't be user-defined?
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
```

Comment:
This is the XC potential, something distinct from the energy. This is already defined in the DFT name.